swan-lake/resources/featured-scenarios/call-llms-with-natural-expressions.md

```diff
@@ -0,0 +1,287 @@
+---
+layout: ballerina-calling-llms-with-natural-expressions-left-nav-pages-swanlake
+title: Call Large Language Models (LLMs) with Natural Expressions 
+description: Bring AI-powered capabilities into your integrations using natural expressions to call LLMs with automatic binding of the response to user-defined types.
+keywords: ballerina, AI, LLM, large language model
+permalink: /learn/call-llms-with-natural-expressions/
+active: call-llms-with-natural-expressions
+intro: This guide helps you understand the basics of Ballerina natural expressions.
+---
+
+## Set up the prerequisites
+
+To complete this tutorial, you need:
+
+1. Ballerina 2201.13.0-m1 (Swan Lake) or greater. Install [Ballerina](/downloads/) and use the `bal dist pull` command to pull the milestone version.
```

Comment:
Don't we need to mention that, if the distribution is already there, use the `bal dist use` command?

Reply:
IMO, if the user has multiple versions, that bit is self-explanatory. Only pulling the milestone version is something new here.